### DISCUSSION TITLE
请问用docker配置的话，怎么可以设置访问固定目录？不是根目录

### DISCUSSION THREAD

**Discussion by water728** [messense/aliyundrive-webdav#433] [Q&A] [answered]
如题。ipk模式可以设置，docker不会，也找不到，谢谢了

**Comment by messense** [comment, marked as answer]
https://github.com/messense/aliyundrive-webdav/discussions/364#discussioncomment-2490951

**Comment by lanyucomputer** [comment]
docker run -d --name=aliyundrive-webdav --restart=unless-stopped \
  -v /etc/aliyundrive-webdav/:/etc/aliyundrive-webdav/ \
  -p 8080:8080 \
  -e REFRESH_TOKEN='your refresh token' \
  -e WEBDAV_AUTH_USER=admin \
  -e WEBDAV_AUTH_PASSWORD=admin \
  messense/aliyundrive-webdav \
  /usr/bin/aliyundrive-webdav --host 0.0.0.0 --auto-index --workdir /etc/aliyundrive-webdav --port 8080 --root /
